modules/gltf/test/gltf-writer.spec.js

```diff
@@ -17,6 +17,8 @@ const REQUIRED_EXTENSION_2 = 'UBER_extension_2';
 const USED_EXTENSION_1 = 'UBER_extension_3';
 const USED_EXTENSION_2 = 'UBER_extension_4';
 
+const GLTF_BINARY_URL = '@loaders.gl/gltf/test/data/gltf-2.0/2CylinderEngine.glb';
```

Comment:
We need to fix the import on line 9 (see comment below) I must have missed that in previous PR.